ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -6,6 +6,13 @@
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "internal-update-case-summary",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRUD"
```

Comment:
vv minor you shouldn't need to give `U` and `D` permissions with events going by [this confluence page](https://tools.hmcts.net/confluence/display/RCCD/Using+CRUD+to+hide+Events+triggering+from+UI) which makes sense